modules/porous_flow/doc/content/source/postprocessors/FVPorousFlowFluidMass.md

```diff
@@ -0,0 +1,28 @@
+# FVPorousFlowFluidMass
+
+!syntax description /Postprocessors/FVPorousFlowFluidMass
+
+This finite volume specific `Postprocessor` calculates the mass of a fluid component $\kappa$ using
+\begin{equation*}
+M^{\kappa} = \phi \sum_{\beta} \chi^{\kappa}_{\beta} \rho_{\beta} S_{\beta},
+\end{equation*}
+where all variables are defined in [`nomenclature`](/nomenclature.md).
+
+The fluid component $\kappa$ is specified in the input parameter `fluid_component`.
+By default, the mass of fluid component $\kappa$ is summed over all fluid phases. The
+sum can be restricted to only a subset of fluid phases by entering the phase indexes
+in the `phase` input parameter.
+
+This `Postprocessor` also provides the option to only calculate fluid mass below a
+certain saturation, which can be invoked using the `saturation_threshold` parameter.
+
+The flag `use_displaced_mesh = false` is set internally by this Postprocessor, and the parameter cannot be altered by the user, even for simulations with solid-mechanical deformation.  The reason is that this Postprocessor uses the strain calculated by TensorMechanics to automatically compensate for deformed meshes.  Further information may be found [here](porous_flow/time_derivative.md).  Therefore:
+
+- For mechanically-coupled simulations, you must provide a `base_name` that is identical to that used by the TensorMechanics strain calculator, so that this Postprocessor can retrieve the correct strain.  The most common use-case is that you provide no `base_name` to the TensorMechanics strain calculator and hence no `base_name` to this Postprocessor.
```

Comment:
```suggestion
- For mechanically-coupled simulations, you must provide a [!param](/Postprocessors/FVPorousFlowFluidMass/base_name) parameter that is identical to that used by the TensorMechanics strain calculator, so that this Postprocessor can retrieve the correct strain.  The most common use-case is that you provide no `base_name` to the TensorMechanics strain calculator and hence no `base_name` to this Postprocessor.
```

Reply:
once is enough to make sure it doesnt change silently breaking the docs imo